roles/cs.php-tideways/defaults/main.yml

```diff
@@ -0,0 +1,32 @@
+tideways_enable: no
```

Comment:
Blackfire has a default of yes. To keep consistency we shall have yes here also, and no in global vars or all.yml